packages/mobile/src/redux/migrations.ts

```diff
@@ -242,4 +243,16 @@ export const migrations = {
       },
     },
   }),
+  15: (state: any) => ({
+    ...state,
+    localCurrency: {
+      ...state.localCurrency,
+      exchangeRate: undefined,
+      exchangeRates: {
+        [Currency.Dollar]: state.localCurrency.exchangeRate,
+        [Currency.Euro]: null,
+        [Currency.Celo]: null,
+      },
+    },
+  }),
```

Comment:
The `stableToken` reducer doesn't handle the rehydrate action and the balances are always refetched on app open, so I don't think we need one

Reply:
See previous comment about the `REHYDRATE` action.
We should be extra careful with all persisted state to avoid crashes or incorrect behaviors. And avoid leaving outdated props for some of our users.
Though unlikely, if the network connection is down the refetch of the balance won't work and the user would be left with empty values.

Reply:
I think showing stale balances with no indication to users is not great UX-wise and we should be able to handle `null` balances without crashing but I see your point. Added the migration.